deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
+    "secret":  "put a secret string here",
+    "noweb":  false,
+    "site":  "My pump.io site using Docker",
+    "owner":  "My name is Docker",
+    "ownerURL":  "http://dockerio.cloudapp.net",
+    "hostname":  "pumpio.example.com",
+    "port":  443,
+    "urlPort": 443,
+    "key": "/etc/ssl/private/privkey.pem",
+    "cert": "/etc/ssl/private/fullchain.pem",
+    "address":  "0.0.0.0",
+    "nologger":  false,
+    "serverUser":  "pumpio",
+    "datadir": "/data/uploads",
```

Comment:
The datadir needs to be persistent, right? That is the reason why I would put it in an unusual location. People tend to think that everything what is in the standard hierarchy will be part of the image and thus destroyed together with the image. But I don't have a strong opinion on that.